webpack.config.js

```diff
@@ -3,10 +3,17 @@
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var ReactStylePlugin = require('react-style-webpack-plugin');
 
+var path = require('path');
+
+var port = JSON.parse(process.env.npm_package_config_port || 6000);
+
 
 module.exports = {
 	devtool: 'sourcemap',
-	entry: "./views/DocumentationApplication.js",
+	entry: [
+      'webpack-dev-server/client?http://localhost:' + port,
```

Comment:
better use `--inline` for the webpack-dev-server. It'll add this automatically...